packages/types/contracts/stake-cw20.ts

```diff
@@ -3,4 +3,5 @@ import { Expiration, Uint128 } from './common'
 export interface Claim {
   amount: Uint128
   release_at: Expiration
+  token_id?: string
```

Comment:
it is probably cleaner to make two different `Claim` types, renaming this to `Cw20Claim` and making another separate `Cw721Claim`

in fact i see `NftClaim` in the `CwdVotingCw721Staked.ts` types file in this same folder; that is probably what we want to use.